src/components/DisplayAddress.tsx

```diff
@@ -1,8 +1,10 @@
 /* eslint-disable react/destructuring-assignment */
 import React from 'react';
 import clsx from 'clsx';
-import { chunkString } from '../utils/basicHelpers';
+import { chunkString } from '../utils/basicHelpers.js';
 
+// Seems bugged?
+// eslint-disable-next-line no-shadow
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/issues/2483
I'm not sure how we avoided this before, but please fix instead of ignoring :shrug:

Reply:
I don't even know why it claims it's shadowing something.. Will see if I can fix it.

Reply:
You fix it by changing the rule, check the link :upside_down_face: